Repository/IRepository.cs

```diff
@@ -0,0 +1,75 @@
+using Microsoft.Azure.Cosmos.Table;
+using System.Threading.Tasks;
+
+namespace NosAyudamos
+{
+    public interface IRepository<T>
+        where T : class, ITableEntity
+    {
+        Task<T> AddOrUpdateAsync(T entity);
```

Comment:
Never liked these too long names... How about making this REST-y? Call it `PutAsync`? Would be consistent with `Delete/Get` async too. `PostAsync` too if you prefer?

Reply:
Should be Put if its add/update